src/universal/modules/patterns/components/Example/Example.js

```diff
@@ -1,43 +1,41 @@
-import React, { Component, PropTypes } from 'react';
-import look, { StyleSheet } from 'react-look';
-
-let styles = {};
-
-@look
-// eslint-disable-next-line react/prefer-stateless-function
-export default class Example extends Component {
-  static propTypes = {
-    children: PropTypes.any
-  }
-
-  render() {
-    return (
-      <div className={styles.base}>
-        <h3 className={styles.heading}>Example</h3>
-        <div className={styles.inner}>
-          {this.props.children}
-        </div>
-      </div>
-    );
-  }
-}
-
-styles = StyleSheet.create({
-  base: {
-    margin: '2rem auto'
-  },
-
-  heading: {
-    backgroundColor: '#999',
-    color: 'white',
-    fontSize: '.875rem',
-    margin: 0,
-    padding: '.5rem 1rem',
-    textTransform: 'uppercase'
-  },
-
-  inner: {
-    border: '1px solid #999',
-    padding: '2rem'
-  }
-});
+// import React, { Component, PropTypes } from 'react';
```

Comment:
I don’t mind yanking the components in patterns. I do think the shortcuts menu is still something we want to work out, perhaps move that to `/meeting` module instead of `/team` module (does that module even get used?)